pkg/status/status.go

```diff
@@ -0,0 +1,126 @@
+package status
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"text/template"
+
+	"github.com/stripe/stripe-cli/pkg/ansi"
+)
+
+// Response contains the structure of system statuses from Stripe
+type Response struct {
+	Statuses    statuses `json:"statuses"`
+	LargeStatus string   `json:"largestatus"`
+	Message     string   `json:"message"`
+	Time        string   `json:"time"`
+}
+
+type statuses struct {
+	API        string `json:"api"`
+	Dashboard  string `json:"dashboard"`
+	Stripejs   string `json:"stripejs"`
+	Checkoutjs string `json:"checkoutjs"`
+	Webhooks   string `json:"webhooks"`
+	Emails     string `json:"emails"`
```

Comment:
Not sure if intentional or not, but "emails" doesn't seem to appear in the formatted output below.

Reply:
I intentionally left it out. We don't report emails on status.stripe.com so I didn't want to cause confusion. Maybe I should remove it from here too?

Reply:
Probably not a big deal either way. If you leave it in, I'd add a comment indicating that it's not currently exposed (and may be not reliable? not sure if there's a reason it's not).